Description: Debianize
 Use files from the fonts-roboto-fontface package.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-07

--- python-xstatic-roboto-fontface-0.4.3.2.orig/xstatic/pkg/roboto_fontface/__init__.py
+++ python-xstatic-roboto-fontface-0.4.3.2/xstatic/pkg/roboto_fontface/__init__.py
@@ -38,9 +38,9 @@ HOMEPAGE = 'https://github.com/choffmeis
 LICENSE = '(same as %s)' % DISPLAY_NAME
 
 from os.path import join, dirname
-BASE_DIR = join(dirname(__file__), 'data')
+#BASE_DIR = join(dirname(__file__), 'data')
 # linux package maintainers just can point to their file locations like this:
-#BASE_DIR = '/usr/share/javascript/' + NAME
+BASE_DIR = '/usr/share/fonts-roboto-fontface'
 
 # location of the Javascript file that's the entry point for this package, if
 # one exists, relative to BASE_DIR
